### ISSUE TITLE
SerializationException with ECS Fargate Tasks

### ISSUE THREAD

**Issue by gsoyka** [containous/traefik#3758]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Attempted to run traefik using the ecs backend.  We are currently utilizing a mix of Fargate and EC2 tasks.  Verified that traefik starts up normally when only EC2 tasks are present, but fails when any Fargate tasks exist within the cluster.

### What did you expect to see?

Traefik starting normally.

### What did you see instead?

```
Provider connection error SerializationException:
	status code: 400, request id: 279485fa-9f13-11e8-a3c7-9510e5251df9, retrying in 714.423969ms
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.6.5
Codename:     tetedemoine
Go version:   go1.10.3
Built:        2018-07-10_03:54:03PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```
docker run -it -p 80:80 -p 8080:8080 traefik:alpine --api --ping --ping.entrypoint=http --ecs --ecs.autodiscoverclusters --ecs.region=us-east-1
```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```
DEBU[2018-08-13T16:12:28Z] Global configuration loaded {"LifeCycle":{"RequestAcceptGraceTimeout":0,"GraceTimeOut":10000000000},"GraceTimeOut":0,"Debug":false,"CheckNewVersion":true,"SendAnonymousUsage":false,"AccessLogsFile":"","AccessLog":null,"TraefikLogsFile":"","TraefikLog":null,"Tracing":null,"LogLevel":"DEBUG","EntryPoints":{"http":{"Address":":80","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"WhiteList":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}},"traefik":{"Address":":8080","TLS":null,"Redirect":null,"Auth":null,"WhitelistSourceRange":null,"WhiteList":null,"Compress":false,"ProxyProtocol":null,"ForwardedHeaders":{"Insecure":true,"TrustedIPs":null}}},"Cluster":null,"Constraints":[],"ACME":null,"DefaultEntryPoints":["http"],"ProvidersThrottleDuration":2000000000,"MaxIdleConnsPerHost":200,"IdleTimeout":0,"InsecureSkipVerify":false,"RootCAs":null,"Retry":null,"HealthCheck":{"Interval":30000000000},"RespondingTimeouts":null,"ForwardingTimeouts":null,"AllowMinWeightZero":false,"Web":null,"Docker":null,"File":null,"Marathon":null,"Consul":null,"ConsulCatalog":null,"Etcd":null,"Zookeeper":null,"Boltdb":null,"Kubernetes":null,"Mesos":null,"Eureka":null,"ECS":{"Watch":true,"Filename":"","Constraints":[],"Trace":false,"TemplateVersion":2,"DebugLogGeneratedTemplate":false,"Domain":"","ExposedByDefault":true,"RefreshSeconds":15,"Clusters":["default"],"Cluster":"","AutoDiscoverClusters":true,"Region":"us-east-1","AccessKeyID":"","SecretAccessKey":""},"Rancher":null,"DynamoDB":null,"ServiceFabric":null,"Rest":null,"API":{"EntryPoint":"traefik","Dashboard":true,"Debug":false,"CurrentConfigurations":null,"Statistics":null},"Metrics":null,"Ping":{"EntryPoint":"http"}}
INFO[2018-08-13T16:12:28Z] Preparing server http &{Address::80 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] WhiteList:<nil> Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202b7760} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s
INFO[2018-08-13T16:12:28Z] Preparing server traefik &{Address::8080 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] WhiteList:<nil> Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc4202b7780} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s
INFO[2018-08-13T16:12:28Z] Starting provider *ecs.Provider {"Watch":true,"Filename":"","Constraints":[],"Trace":false,"TemplateVersion":2,"DebugLogGeneratedTemplate":false,"Domain":"","ExposedByDefault":true,"RefreshSeconds":15,"Clusters":["default"],"Cluster":"","AutoDiscoverClusters":true,"Region":"us-east-1","AccessKeyID":"","SecretAccessKey":""}
INFO[2018-08-13T16:12:28Z] Starting server on :80
INFO[2018-08-13T16:12:28Z] Starting server on :8080
DEBU[2018-08-13T16:12:28Z] ECS Clusters: [arn:aws:ecs:us-east-1:632106808859:cluster/ecs-cluster-Cluster-9DEW5N7BJ2IC]
2018/08/13 16:12:28 Request.NextPage deprecated. Use Pagination type for configurable pagination of API operations
ERRO[2018-08-13T16:12:28Z] Provider connection error SerializationException:
	status code: 400, request id: ace788e9-9f13-11e8-8e95-a9927097b410, retrying in 286.075265ms
DEBU[2018-08-13T16:12:29Z] ECS Clusters: [arn:aws:ecs:us-east-1:632106808859:cluster/ecs-cluster-Cluster-9DEW5N7BJ2IC]
```

**Comment by mmatur** [comment]
Hi @gsoyka,

Thanks for your interest in Træfik.

I think your issue is fixed by https://github.com/containous/traefik/pull/3379.

The fargate support has been added in Traefik v1.7. It is not available in v1.6.

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of tidying up the issues.

I have to close this one because of its lack of activity :disappointed:

Feel free to re-open it or join our [Slack workspace](https://traefik.herokuapp.com/) for more [community #support](https://traefik.slack.com/messages/C0CDT22PJ/).
